main.cpp

```diff
@@ -170,31 +169,27 @@ struct DoubleType
     DoubleType& powInternal(double myNumber);
 };
 
-DoubleType& DoubleType::add(double myNumber)
+DoubleType& DoubleType::operator+=(double myNumber)
 {
     *value += myNumber;
     return *this;
 }
 
-DoubleType& DoubleType::subtract(double myNumber)
+DoubleType& DoubleType::operator-=(double myNumber)
 {
     *value -= myNumber;
     return *this;
 }
 
-DoubleType& DoubleType::multiply(double myNumber)
+DoubleType& DoubleType::operator*=(double myNumber)
 {
-    *value *= myNumber;
+    *value /= myNumber;
     return *this;
 }
 
-DoubleType& DoubleType::divide(double myNumber)
+DoubleType& DoubleType::operator/=(double myNumber)
 {
-    if (myNumber == 0.)
-    {
-        std::cout << "You're dividing by double 0!" << std::endl;
-    }
-    *value /= myNumber; 
+    *value /= myNumber;
     return *this;
```

Comment:
Just because you change to operator/= doesn't mean the functionality should change.   your divide/assign function body should stay the same.

- [ ] revert to the body with the warning for your `operator/=`